### ISSUE TITLE
convert msgpack ctraces to JSON generate invalid characters

### ISSUE THREAD

**Issue by edsiper** [fluent/fluent-bit#6810]
I am generating a dummy trace in Fluent Bit with ctrace api, but when converting the msgpack version to JSON I get invalid characters, specifically around `trace_id`, `span_id`.. this is not expected:

```
{"resourceSpans":[{"resource":{"attributes":{},"dropped_attributes_count":5},"schema_url":"https://ctraces/resource_span_schema_url","scope_spans":[{"scope":{"name":"ctrace","version":"a.b.c","attributes":null,"dropped_attributes_count":3},"spans":[{"trace_id":"^M 2C\\X@","span_id":"SL\u007f","parent_span_id":null,"trace_state":null,"name":"main","kind":1,"start_time_unix_nano":1675883312238577757,"end_time_unix_nano":1675883312238577757,"attributes":{"agent":"Fluent Bit","year":2022,"open_source":true,"temperature":25.5,"my_array":["first",2.0,false,[3.1,5.2,6.3]],"my-list":{"language":"c"}},"dropped_attributes_count":0,"events":[{"time_unix_nano":1675883312238600048,"name":"connect to remote server","attributes":{"syscall 1":"open()","syscall 2":"connect()","syscall 3":"write()"},"dropped_attributes_count":0}],"links":[],"status":{"code":0,"message":null}},{"trace_id":"^M 2C\\X@","span_id":"K\u0011өNkk","parent_span_id":"SL\u007f","trace_state":null,"name":"do-work","kind":3,"start_time_unix_nano":1675883312238607548,"end_time_unix_nano":1675883312238607548,"attributes":{},"dropped_attributes_count":0,"events":[],"links":[{"trace_id":"RrN1n@:\u001d`","span_id":"\u0019","trace_state":"aaabbbccc","attributes":null,"dropped_attributes_count":2}],"status":{"code":0,"message":null}}],"schema_url":"https://ctraces/scope_span_schema_url"}]}]}
```

**repro**

in an output plugin like stdout, convert msgpack trace to JSON:

```
pack = flb_msgpack_raw_to_json_sds(event_chunk->data, event_chunk->size);
printf("%s\n", pack);
```

**Comment by edsiper** [comment]
test: 

`fluent-bit -i event_type -p type=traces -o ....`

**Comment by edsiper** [comment]
fixed by upgrading to CTraces v0.3.0.
